static/js/main.js

```diff
@@ -75,6 +75,7 @@ var monito = (function (preferences, requests, gui) {
         init: init,
         refresh: autoRefresh,
         displayInfo: displayInfo,
-        displayMenu: displayMenu
+        displayMenu: displayMenu,
+        showSnackbar: _helpers.showSnackbar
     }
 })(_prefs, _requests, _gui);
```

Comment:
Pass in the _helpers here in the method call otherwise it won't be accessible inside.